command/connect/envoy/fdutil/fdutil.go

```diff
@@ -0,0 +1,43 @@
+// package fdutil provides some utilities to manipulate file descriptors
+package fdutil
+
+import (
+	"golang.org/x/sys/unix"
+)
```

Comment:
This will fail our build when we try to cross-compile, can we add build tags to only compile on unix and add dummy implementations for other OSes that just error?